/*
  Device Vehicle Mapping

  This section holds messages used in self-mapping service.

  The version of the OpenAPI document: 2

  Generated by OpenAPI Generator: https://openapi-generator.tech
*/

syntax = "proto3";

package com.bosch.bfb.protobuf.devicevehiclemapping.v2;

import public "device_to_backend_message_meta.proto";

message VinUpdateEvent {

  DeviceToBackendMessageMeta meta = 3347973;

  // The VIN that was detected by the device. Empty string indicates removal of device from vehicle.
  string vin = 116763;

  // Whether or not the detected VIN is trusted.
  bool vinIsTrusted = 341597552;

}
